run.py

```diff
@@ -84,11 +99,55 @@ def main():
     tasks = get_config(yaml_path)
    # Validates YAML File
     validator(tasks)
-    for test in args.test:
-        if test == "storage":
-            #passes the parameters loaded from yaml file to helper function
-            run_storage(tasks)
-            cleanup()
+
+    if (args.user_mode):
+        if(args.session == None):
+            raise Exception("session argument needed")
+        for test in args.test:
+            if test == "storage":
+                storage_user_container(args.session)
+                """
+                Run the file run_container.py
+                """
+                command = "sudo docker exec -it -u "+args.session+" -w /scratch/" + args.session + " " + "jupyter-" + args.session + " " + "python3 run_container.py --test " + test
+                # client.containers.get("jupyter-user2").exec_run(cmd = ["python3", "run_container.py"], user = "user2", stdin=False, workdir= "/scratch/user2/", stream = True, stdout=True)
```

Comment:
I could have used docker python API to exec into the container but then we would not be able to see the logs which are printed on the terminal , even after setting the stdout to true. However the output of the statement client.containers.get("jupyter-user2").exec_run(....) can be stored in a variable then we can print (pretty print) that variable to see the logs on the terminal.